src/sdk/models/categories.ts

```diff
@@ -0,0 +1,5 @@
+import { IEntityIdentifier } from './base';
+
+export interface Category extends IEntityIdentifier<string> {
```

Comment:
`Category` is too generic since we also have `JobCategory` and potentially others down the road. I would change the class and resource names to be prefixed with `Audit`. So, `AuditCategory` and `AuditCategoriesResource`. I would also change the command to `audit-categories`